tests/unit/test-cmd/test.build.js

```diff
@@ -5,11 +5,37 @@ import {it, describe} from 'mocha';
 import {assert} from 'chai';
 import sinon from 'sinon';
 
-import build, {safeFileName, FileFilter} from '../../../src/cmd/build';
+import build, {safeFileName, FileFilter,
+                getDefaultLocalizedName} from '../../../src/cmd/build';
 import {withTempDir} from '../../../src/util/temp-dir';
 import {fixturePath, makeSureItFails, ZipFile} from '../helpers';
-import {basicManifest, manifestWithoutApps} from '../test-util/test.manifest';
+import {basicManifest,
+  manifestWithoutApps,
+} from '../test-util/test.manifest';
+import {WebExtError} from '../../../src/errors';
+import {createLogger} from '../../../src/util/logger';
 
+const basicLocalizedManifest = {
+  name: '__MSG_extensionName__',
+  version: '0.0.1',
+  applications: {
```

Comment:
you can remove the `applications` part since it's not required and not needed for the test